### ISSUE TITLE
Show What Other Users Are Playing

### ISSUE THREAD

**Issue by ghost** [navidrome/navidrome#769]
It is very nice to see what other users are playing on the server.  Perhaps as a list on the left pane?

**Comment by SteveDinn** [comment]
I think this should be left to third parties like Last FM. It adds complexity where it doesn't belong

**Comment by deluan** [comment]
You mean show what is currently being played? If that is the case, then yes, Navidrome already implement the [getNowPlaying](http://subsonic.org/pages/api.jsp#getNowPlaying) Subsonic endpoint, and the plan is to show this info in the Activity Panel.

**Comment by TheFlatworm** [comment]
"You mean show what is currently being played?"

Yes, including other users on the server, so it show "USER1 is listening to Bach Brandenburg #1", "USER2 is listening to the Brand Flakes, Porky Pig" etc... much like the Subsonic feature.

**Comment by parthsarthiprasad** [comment]
@deluan do you mean something like a story icon, fetching data from users in a friend list?

**Comment by deluan** [comment]
@parthsarthiprasad If you look at the [getNowPlaying](http://subsonic.org/pages/api.jsp#getNowPlaying) response, you'll see what information you have. The idea is to add this information to the Activity Panel:

<img width="392" alt="Screen Shot 2021-03-11 at 10 57 19 AM" src="https://user-images.githubusercontent.com/331353/110815616-998fc080-8258-11eb-82f3-8245f6bec060.png">

This is how it is implemented in Subsonic itself (you can see it working in their [demo server](http://subsonic.org/pages/demo.jsp): 
<img width="285" alt="Screen Shot 2021-03-11 at 10 58 17 AM" src="https://user-images.githubusercontent.com/331353/110815782-bfb56080-8258-11eb-9376-3f19cb90eb81.png">

**Comment by Rishabh-malhotraa** [comment]
@deluan 
this is the response i get from the supersonic API
And I want to display this information like below )

Having the cover-art on the left and the name of the user on the right 

![Screenshot from 2021-03-29 01-20-26](https://user-images.githubusercontent.com/54576074/112765898-fb0fa780-902c-11eb-8a03-8cf55dc545bc.png)


we could show the information about the song as a tooltip like a popover, or maybe just show the name of the song with the help of a tooltip like this 


https://user-images.githubusercontent.com/54576074/112765875-e3382380-902c-11eb-9bdf-e169a49e7528.mp4




**also open to feedback if you want this implemented in a different way**

``` js
// the response from /rest/getNowPlaying
[
  { 
     "album": "Hollywood's Bleeding",
     "albumId": "279c18e00d359ec0da3953d6179b744c",
     "artist": "Post Malone; Louis Bell; Brian Lee",
     "artistId": "483fa419fb72bbc25ed1f7e24097ee72",
     "bitRate": 320,
     "contentType": "audio/mpeg",
     "coverArt": "58fb2e2f97f650dedd3365e71632a594",
     "created": "2021-03-27T13:37:11.11628291Z",
     "discNumber": 1,
     "duration": 156,
     "genre": "Rap/Hip Hop",
     "id": "58fb2e2f97f650dedd3365e71632a594",
     "isDir": false,
     "isVideo": false,
     "minutesAgo": 6,
     "parent": "279c18e00d359ec0da3953d6179b744c",
     "path": "Post Malone/Hollywood's Bleeding/Hollywood's Bleeding.mp3",,
     "playerId": 1,
     "playerName": "NavidromeUI",
     "size": 6310644,
     "suffix": "mp3",
     "title": "Hollywood's Bleeding",
     "track": 1,
     "type": "music",
     "username": "admin",
     "year": 2019
  }
]
```

**The part I need help with is the cover-art in the response just seems a random string which I cannot make sense of, so how can you extract the image information with that.**

**Comment by deluan** [comment]
> Did some further digging on how images are fetched and it can be fetched using `getCoverArt/?id=` and passing the id of the required song

Yes, there is a `subsonic` "client" in the UI code, that you can call: `subsonic.getCoverArtUrl(record, size)`

I think it should show (without any popover/tooltip): 
- username
- artist
- title
- small coverArt (with a link to the album, similar to what we have in the player)

**Comment by Rishabh-malhotraa** [comment]
This is the way I have currently made it:


![Screenshot from 2021-04-03 00-30-01](https://user-images.githubusercontent.com/54576074/113445607-cf693480-9413-11eb-8f9f-0b8bbf4405c9.png)


**In the following I am not rendering the album all the information about the song but I feel it looks a bit cleaner**


![Screenshot from 2021-04-03 00-32-15](https://user-images.githubusercontent.com/54576074/113445824-31299e80-9414-11eb-8c92-c730d079573f.png)

Do let me know  which one should i go ahead with or do something different alltogether

**Comment by deluan** [comment]
Implemented in #769
